common/src/IO/LoadTextureCollection.cpp

```diff
@@ -88,8 +90,33 @@ Result<Assets::Texture, ReadTextureError> readTexture(
   const size_t prefixLength,
   const std::optional<Assets::Palette>& palette)
 {
-  static const auto imageFileExtensions =
-    std::vector<std::string>{".jpg", ".jpeg", ".png", ".tga", ".bmp"};
+  static std::vector<std::string> imageFileExtensions;
+  if (imageFileExtensions.empty())
```

Comment:
This should be moved into a separate private function in an anonymous namespace, then you can write 

```
static const auto imageFileExtensions = getSupportedFreeImageExtensions();
```